### PR TITLE
simple advanced panels

### PR DESCRIPTION
**Pull request by wangmingliang-ms**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/webapp/WebAppService.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import com.microsoft.azuretools.telemetrywrapper.*;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Map;
+
+import static com.microsoft.azuretools.telemetry.TelemetryConstants.CREATE_WEBAPP;
+import static com.microsoft.azuretools.telemetry.TelemetryConstants.WEBAPP;
+
+public class WebAppService {
+    public static WebAppService instance = new WebAppService();
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/webapp/WebAppService.java:17]: private

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/webapp/WebAppService.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        final WebAppSettingModel settings = convertConfig2Settings(config);
+        settings.setCreatingNew(true);
+        final Map<String, String> properties = settings.getTelemetryProperties(null);
+        final Operation operation = TelemetryManager.createOperation(WEBAPP, CREATE_WEBAPP);
+        try {
+            operation.start();
+            EventUtil.logEvent(EventType.info, operation, properties);
+            return AzureWebAppMvpModel.getInstance().createWebApp(settings);
+        } catch (final Exception e) {
+            EventUtil.logError(operation, ErrorType.userError, e, properties, null);
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/webapp/WebAppService.java:33]: why this exception is user error?

Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/webapp/WebAppService.java:33]: 😢 copied from original `WebAppCreationDialg`

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/intellij/view/AppServiceConfigFormPanelAdvanced.java [hunk 3]
```diff
@@ -1,52 +1,55 @@
...
-    val tableName = args(1)
-    val sqlStatement = args(2)
+    protected JTextField textName;
+    protected ComboBoxPlatform selectorRuntime;
+    protected JBRadioButton radioOsLinux;
+    protected JBRadioButton radioOsWindows;
+    protected ComboBoxRegion selectorRegion;
 
-    val df = spark.read.json(json)
+    protected JLabel textSku;
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/intellij/view/AppServiceConfigFormPanelAdvanced.java:42]: protected -> private

**Pull request merged**